src/Common/Constants.ts

```diff
@@ -354,6 +354,10 @@ export enum ContainerStatusType {
   Disconnected = "Disconnected",
 }
 
+export enum PoolIdType {
```

Comment:
Shouldn't this be PoolIds?

Reply:
For now we are not sure if we want to have per feature a poolid or multiple poolid's per feature. So going with PoolIdType